evalml/preprocessing/utils.py

```diff
@@ -79,7 +79,7 @@ def split_multiseries_data(X, y, series_id, time_index, **kwargs):
 
     # Get unique series value from X if there is only the time_index column
     # Otherwise, this information is generated in `stack_X` from the column values
-    series_id_values = set(X[series_id]) if len(X_unstacked.columns) == 1 else None
+    series_id_values = X[series_id].unique() if len(X_unstacked.columns) == 1 else None
```

Comment:
What's the difference with this change? Is it just list vs set data format?

Reply:
This was done to maintain the input series ID order. I can add a clarifying comment here.